packages/playwright/src/program.ts

```diff
@@ -332,9 +332,34 @@ function resolveReporter(id: string) {
   return require.resolve(id, { paths: [process.cwd()] });
 }
 
+class OptionValidators {
+  static validateWorkers(value: string): string {
+    if (value.endsWith('%'))
+      return value;
```

Comment:
Let's validate it's not `abc%`? While you are here, might as well similarly fix this place: https://github.com/microsoft/playwright/blob/main/packages/playwright/src/common/config.ts#L106C62-L106C83.